docs/development/development-environment/docker/README.md

```diff
@@ -384,15 +387,15 @@ TLS support. Basically, you need to tell `traefik` for which docker compose serv
 There is an example compose file (see `docker/dev/tls/docker-compose.core-override.example.yml`), which contents you can
 take over to your custom `docker-compose.override.yml` in the repository root.
 
-In addition, we need to alter the environmental variables used in the new overrides. So we need to amend the `.env` file
+In addition, we need to alter the environmental variables used in the new overrides. So we need to amend the `.env.development.local` file
 like that:
 
 ```shell
 OPENPROJECT_DEV_HOST=openproject.local
 OPENPROJECT_DEV_URL=https://${OPENPROJECT_DEV_HOST}
 ```
 
-After amending the override file and the `.env`, ensure that you restart the stack.
+After amending the override file and the `.env.development.local`, ensure that you restart the stack.
```

Comment:
TBD: I just noticed that my local web worker setup is borked. Not sure if my changes here are the cause.

Reply:
I think my suggestion to move this stuff to `env.development.local` is entirely wrong. The problem I had was that I overwrite `OPENPROJECT_HOST_NAME` as well, which will then break tests.

However, I can't find any instructions to do that, so I am wondering why I did it in the first place. As far as I can tell, by just following the original instructions here, both tests and web server should work fine.